ci_feature/Dockerfile

```diff
@@ -5,6 +5,7 @@ com.microsoft.product="OMS Container Docker Provider" \
 com.microsoft.version="2.0.0-7"
 ENV tmpdir /opt
 ENV APPLICATIONINSIGHTS_AUTH OTQzNWI0M2YtOTdkNS00ZGVkLThkOTAtYjA0Nzk1OGU2ZTg3
+ENV AGENT_VERSION containerinventory-cifeature-test-2
```

Comment:
can u also add this to ci_feature_prod folder Dockerfile in ci_feature ?

Reply:
sure